Userland/Utilities/drain.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2023, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibFileSystem/FileSystem.h>
+#include <LibMain/Main.h>
+
+constexpr size_t BLOCK_SIZE = 16 * 16384;
```

Comment:
Let's make this readable?
```suggestion
constexpr size_t BLOCK_SIZE = 256 * KiB;
```

(p.s. T-T don't scream at me, constant)